R/SSplotSPR.R

```diff
@@ -45,7 +48,7 @@ SSplotSPR <-
            uncertainty = TRUE,
            subplots = 1:4, forecastplot = FALSE,
            col1 = "black", col2 = "blue", col3 = "green3", col4 = "red",
-           sprtarg = "default", btarg = "default",
+           sprtarg = "default", btarg = "default", minbthresh = "default",
```

Comment:
I wonder if this should instead be `minbthresh = replist[["minbthresh"]]`, as I think making defaults directly assigned when they are simple enough is supposed to be a good practice? Then there is no need to explain what default means in the documentation.

Reply:
I will open a separate issue to apply this to other functions.